router/rulerouter/relay.go

```diff
@@ -63,6 +65,8 @@ type RelayStateImpl struct {
 	WorldShardFallback bool
 	routerMode         config.RouterMode
 
+	extendedDebug bool
```

Comment:
Why you just don't getting the setting from router's config?

Reply:
> Why you just don't get the setting from router's config?

because of reload